test/parallel/test-assert.js

```diff
@@ -958,7 +958,9 @@ common.expectsError(
     }
   );
 
-  assert.throws(() => { throw new Error('e'); }, new Error('e'));
+  // [browserify] Error objects will not be strictly equal in some browsers due
+  // to different `line`/`column` properties.
+  // assert.throws(() => { throw new Error('e'); }, new Error('e'));
```

Comment:
@lukechilds I am not sure what the comment stands for. The `stack` property should not be inspected. But I imagine some browsers might have the stack as enumerable property?

Reply:
I'm not sure exactly why the behaviour is different in Safari, but it doesn't seem to be the stack property, the properties are directly attached to the error object:

```js
// Chrome
console.log(JSON.stringify(new Error('e')))
{}

// Safari
console.log(JSON.stringify(new Error('e')))
{"line":3,"column":37}
```